src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -145,18 +118,14 @@ func Init(userConfig UserConfig, version string, configDir string, stickyConfig
 		log.Errorf("Could not get config path? %v", err)
 		return nil, err
 	}
-	run := isGoodConfig(configPath)
```

Comment:
@uaalto Until today, I hadn't realized that we were breaking encapsulation and reading/writing the config from within flashlight without using yamlconf.  That scheme obviously broke once we introduced obfuscation.  So, rather than apply obfuscation in both places, I cleaned this up so that everything is handled in yamlconf.